tests/MillisecondTimestampBehaviorTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace ipl\Tests\Orm;
+
+use DateTime;
+use InvalidArgumentException;
+use ipl\Orm\Behavior\MillisecondTimestamp;
+use PHPUnit\Framework\TestCase;
+
+class MillisecondTimestampBehaviorTest extends TestCase
```

Comment:
Please add a test that passes a `DateTime` with a non-UTC timezone and retrieves the result again and expects a `DateTime` that (while UTC is the default timezone) is in UTC and is correct.

Reply:
Oh, and a test that expects (while the default timezone is not UTC) a `DateTime` with the current timezone.